<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutSoftwareDialog</class>
 <widget class="QWidget" name="AboutSoftwareDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/com.wingsummer.winghex/images/icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>WingHexExplorer</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblVersion">
           <property name="text">
            <string notr="true">-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAbout">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QTextBrowser" name="tbAbout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBaseObj">
      <attribute name="title">
       <string>Components</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QTextBrowser" name="tbBaseObj"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLicense">
      <attribute name="title">
       <string>License</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QTextBrowser" name="tbLicense">
         <property name="documentTitle">
          <string/>
         </property>
         <property name="placeholderText">
          <string notr="true"/>
         </property>
         <property name="source">
          <url>
           <string>qrc:/com.wingsummer.winghex/LICENSE</string>
          </url>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDev">
      <attribute name="title">
       <string>Developer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QTextBrowser" name="tbDev"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCredit">
      <attribute name="title">
       <string>Credit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QTextBrowser" name="tbCredit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTr">
      <attribute name="title">
       <string>Translator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QTextBrowser" name="tbTr"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
